src/modules/Touchdown/components/DailyHistory.js

```diff
@@ -0,0 +1,263 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+import { useLazyQuery } from '@apollo/client';

```

Comment:
/* eslint-disable react/prop-types */
/* eslint-disable no-unused-vars */

remove. this and solve errors